diff --git a/.gitignore b/.gitignore
index c465a7b..347d0b8 100644
--- a/.gitignore
+++ b/.gitignore
@@ -68,14 +68,6 @@ typings/
 # Output of 'npm pack'
 *.tgz
 
-# Redundant files
-packages/*/templates/.github/
-packages/*/.npmignore
-packages/*/.drone.yml
-packages/*/templates/CHANGELOG.md.ejs
-packages/*/LICENSE
-packages/generator-ts-project/templates/.gitignore.ejs
-
 # Yarn Integrity file
 .yarn-integrity
 
