diff --git a/.npmignore b/.npmignore
index 493e1e8..74af7d7 100644
--- a/.npmignore
+++ b/.npmignore
@@ -1,6 +1,3 @@
-!.gitignore
-!.npmignore
-
 # Logs
 logs
 *.log
@@ -107,15 +104,26 @@ dist
 .tern-port
 
 # Source-files
-[Ss]rc/**/*
-![Ss]rc/tests/tsconfig.json
+[Ss]rc/
+
+# TypeScript config-files
+tsconfig.json
+tsconfig.*.json
+
+# Lint config-files
+.eslintrc
+.eslintrc.*
 
 # Source-maps
 [Ll]ib/**/*.map
 
 # Unit-Tests
+.mocharc.*
 [Ll]ib/tests/
 
+# Visual Studio Code-Environment
+.vscode/
+
 # GitHub configuration
 .github/
 
